Assets/MixedRealityToolkit.Examples/Demos/StandardShader/Scripts/MaterialParameterChanger.cs

```diff
@@ -0,0 +1,532 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.UI;
+using UnityEngine.UI;
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Examples
+{
+    /// <summary>
+    /// This class changes the MRTKStandardShader parameter of the material attached using PinchSlider.
+    /// By acquiring each method in the event of PinchSlider.sc, you can see the function of MRTKStandardShader on the actual machine.
+    /// </summary>
+    public class MaterialParameterChanger : MonoBehaviour
+    {
+        /// <summary>
+        /// parameter
+        /// </summary>
+        //TargetMateria;
+        public Material material;
+
+
+        //SliderValueText
+        public List<Text> SliderValueTextList;
+        private int x;
+
+        //parameter
+        private float somootheness;
+        private float roundcornermargin;
+        private float roundcornerradius;
+
+        private float hoverRed;
+        private float hoverGreen;
+        private float hoverBlue;
+
+        private float InnerGlowRed;
+        private float InnerGlowGreen;
```

Comment:
I don't believe these are all necessary? They aren't exactly accurate as every instance of the component is duplicating these and some are populated while others aren't.